packages/angular/src/builders/webpack-dev-server/lib/normalize-options.ts

```diff
@@ -1,12 +1,17 @@
+import { homedir } from 'node:os';
 import type { Schema } from '../schema';
 
 export function normalizeOptions(schema: Schema): Schema {
+  const { ssl, sslCert, sslKey } = schema;
+
   return {
     host: 'localhost',
     port: 4200,
     liveReload: true,
     open: false,
-    ssl: false,
     ...schema,
+    ssl: ssl ?? false,
+    ...(sslCert ? { sslCert: sslCert.replace('~', homedir()) } : {}),
+    ...(sslKey ? { sslKey: sslKey.replace('~', homedir()) } : {}),
```

Comment:
Maybe it should be on when it starts with `~`?

Reply:
Let me discuss this PR internally. 

Usually, we only handle files that are within the workspace, specifying the home directory would involve working with files outside the workspace.

Reply:
Thanks! Please let me know 🙏🏼 

I will elaborate on our use case:
We are working with HTTPS and each developer generates his own SSL certificate on his machine, then, it is the same for everyone to use `~` (same for other repositories as well and not only the workspace).

What we are doing now, is some kind of overriding the `webpack-dev-server` but it's making some nonstandard scenarios to handle with + maintain.

Supporting the `~` will let us finally use the `nx serve` command properly as a standard and without any inheritance/overriding it :)